pkg/dashboard/server/flux.go

```diff
@@ -54,6 +58,15 @@ func fluxEvent(w http.ResponseWriter, r *http.Request) {
 		log.Errorf("could not save or update gitops commit: %s", err)
 	}
 
+	gitopsRepoCache := ctx.Value("gitRepoCache").(*nativeGit.RepoCache)
+	perf := ctx.Value("perf").(*prometheus.HistogramVec)
+	err = updateGitopsCommitStatuses(gitopsRepoCache, perf, store, repoName, env, event.Message, repoPerEnv)
+	if err != nil {
+		log.Errorf("cannot update releases: %s", err)
+		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+		return
+	}
+
```

Comment:
Later in the code we broadcast the gitops commit state, the commit that Flux informs us about.
`streaming.BroadcastGitopsCommitEvent(clientHub, *gitopsCommit)`

But the logic has been extended to update the data for commits that Flux did not explicitly informed us about, but instead we know implicitly that they were updated too. Why not stream the state updates of those commits too?